tools/css-modules-v9-codemod.js

```
@@ -0,0 +1,143 @@
+      // For TypeScript: Convert to namespace import (import * as styles)
+      const namespaceSpecifier = j.importNamespaceSpecifier(
+        j.identifier(defaultImportName)
+      );
+      
+      // Replace the import specifiers
+      importDecl.specifiers = [namespaceSpecifier];
+      hasChanges = true;
+      
+      console.log(`✅ [TS] Updated ${fileInfo.path}: import * as ${defaultImportName} from '${sourcePath}'`);
+    } else {
+      // For JavaScript: Convert to named imports
```

ESLint

[error] Insert `··`

(prettier/prettier)